src/HighTable.css

```diff
@@ -225,46 +218,35 @@ tr.selected th:first-child {
 }
 
 /* table corner */
-.table-corner {
+.table thead .table-corner {
   background-color: #f9f4ff;
   border-right: 1px solid #ccc;
-  position: absolute;
-  height: 33px;
-  width: 32px;
-  top: 0;
-  left: 0;
-  z-index: 15;
+  position: sticky;
   box-shadow: inset 0 0 4px rgba(0, 0, 0, 0.2);
   cursor: default;
-  display: flex;
-  justify-content: center;
+  min-width: 32px;
+  max-width: none;
+  width: 32px;
+  z-index: 15;
+  top: -1px; /* fix 1px gap above thead */
+  left: 0;
+  height: 34px;
 }
 /* replace corner with checkbox if selection is enabled (read-only or not) */
-.table-corner input {
+.table thead .table-corner input {
   display: none;
+  margin: 0 auto;
 }
-.table-corner.show-corner-selection  {
-  background: #f9f4ff;
+.table thead .table-corner.show-checkbox {
+  background: #f9f4ff; /* redundant with td.table-corner, but allows to force the background if it has been overridden with a logo */
 }
-.table-corner.show-corner-selection  span {
+.table thead .table-corner.show-checkbox span {
   display: none;
 }
-.table-corner.show-corner-selection  input {
-  display: inline;
-  text-align: center;
+.table thead .table-corner.show-checkbox input {
+  display: block;
 }
 /* if selectable, show pointer cursor on checkbox */
-.selectable .table-corner, .selectable .table-corner input {
+.selectable .table thead .table-corner, .selectable .table thead .table-corner input {
   cursor: pointer;
 }
-
-/* mock row numbers */
-.mock-row-label {
```

Comment:
unless you added this back somewhere else, I think we DO want this mock-row-label. It adds a grey bar that matches the row label color. This way when you make a big scroll down, visually you don't see a flash of white on the side.

Reply:
Oh OK. Indeed, it is needed, I'll restore it.

Reply:
(done in https://github.com/hyparam/hightable/pull/71)